BUILDER_BASE_TAG_FILE

```diff
@@ -1 +1 @@
-standard-8026816b32209021fe189380fb51e25690f08d37.2
\ No newline at end of file
+d2a0e82f228698b9418fac18fed600fda36de814.2
```

Comment:
@jaxesn This is missing the `standard` prefix. Does that matter?

Reply:
Having `standard` here till replace all images with the `standard-` prefix, even ones that were originally using the `minimal` prefix. That's not what we want, so we have just the commit hash here, followed by the Amazon Linux version, and we replace only that part of the image tag, and let the prefixes be untouched by the automation.

Reply:
Nvm, @jaxesn is making a change to re-add the prefix here
aws/eks-distro-build-tooling#686